src/postman/transformers/server.ts

```diff
@@ -0,0 +1,10 @@
+import { IServer } from '@stoplight/types/dist';
+import { Url } from 'postman-collection';
+
+export function transformServer(url: Url): IServer | undefined {
+  if (!url.host) return;
```

Comment:
I'm really wondering why didn't they make this required.

Reply:
According to the spec the host can be a string or an array of strings — is the SDK taking care of this automatically?

Reply:
The SDK is taking care of conversion `string|string[] => string[]` in that case.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/postman-collection/index.d.ts#L616 - that's how `Url` is defined - it's an `string[]`
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/postman-collection/index.d.ts#L605 - `UrlDefinition` - here we deal with `string | string[]`

Ofc since those types are known to have bugs I double-checked it with tests and this time it acts like it was defined.


---

But you spotted another possible bug: protocol might be undefined. I fixed that already.